lms/djangoapps/courseware/grades.py

```diff
@@ -267,6 +268,8 @@ def create_module(descriptor):
 
         totaled_scores[section_format] = format_scores
 
+    # Grading policy might be overriden by a POC, need to reset it
+    course.set_grading_policy(course.grading_policy)
```

Comment:
My discussions with the 15.390.1x course team indicate they want certificates in POCs.  POC instructors don't see the distinction between POCs and MOOCs that rules out the award of certificates.  Neither do I.